### DISCUSSION TITLE
Problems with Quarkus Extension Configuration Annotations (ConfigRoot, ConfigMapping...) used in Application

### DISCUSSION THREAD

**Discussion by anchialas** [quarkusio/quarkus#30606] [Q&A] [answered]
Hello there,

I would like to configure properties in a Quarkus application with the [Quarkus Extension configuration mechanism](https://quarkus.io/guides/writing-extensions#configuration) and generate documentation.

For this I've included the `quarkus-extension-processor`. When I write a simple Config-Object as Java Interface with `@ConfigRoot` and `@ConfigMapping` annotations, it works fine. But with an aggregate configuration object (annotated with `@ConfigGroup`) the asciidoc fragments are generated by the annotation processor, but unfortunately without the description from the javadoc.

![image](https://user-images.githubusercontent.com/2034531/214788566-46f61c99-1812-4bf2-8902-09d99a89ac98.png)

For better illustration I have created an example project:
https://github.com/anchialas/quarkus-app-config

**Comment by radcortez** [comment, marked as answer]
https://github.com/quarkusio/quarkus/pull/32085 will fix it.

**Reply by anchialas** [reply]
That's great, thank you!
